qa/t/rest_github_push_comment.t

```diff
@@ -98,14 +98,21 @@ $t->post_ok(
 $payload = {
   ref        => 'refs/heads/releases_v110',
   repository => {full_name => 'mozilla-mobile/firefox-android'},
+  created    => false,
+  deleted    => false,
+  forced     => false,
   commits    => [{
-    author => {username => 'foobar'},
-    url => 'https://github.com/mozilla-bteam/bmo/commit/abcdefghijklmnopqrstuvwxyz',
-    message => "Bug $bug_id - Test Github Push Comment",
+    author   => {username => 'foobar'},
+    url      => 'https://github.com/mozilla-bteam/bmo/commit/abcdefghijklmnopqrstuvwxyz',
+    message  => "Bug $bug_id - Test Github Push Comment",
```

Comment:
This is out of scope for this PR, but we do parse the first line out of the commit message IIRC. We probably want to add a multi-line commit message to the test data to ensure that parsing is working correctly.

Reply:
Good idea. I will file a bug for that.